README.md

```diff
@@ -1,2 +1,249 @@
 # bootstrap
+
 Foundation Bootstrap Team
+
+## About this document
+
+The Node.js and JS Foundations have announced an intent to merge into a single
+umbrella organization hosting the combined portfolio of projects currently
+under each separate organization.
+
+The Board's of each Foundation have asked the various projects to come together
+and provide specific feedback on:
+
+* Whether the merger should proceed
+* What the overall structure of a merged Foundation should look like
+* How projects should be organized within a merged Foundation
+* What representation projects should have on a merged Foundation's Board of
+  Directors
+* Suggestions for the name of the proposed new Foundation
+
+To help with the process of organizing, drafting, and submitting the requested
+feedback, the technical projects under both the Node.js and JS Foundations are
+forming a temporary body (the "Node+JS Bootstrap Committee").
+
+This document, along with the other files in this repository, are living
+documents where the Committee will capture and prepare the feedback requested
+by the Boards. The documents here will not represent the finalized governance
+model of a merged Foundation but will serve as input into the generation of
+new Foundation Bylaws and Top Level Project Charters should the merger of the
+Foundations move forward.
+
+Until such time that this group has reached consensus on a new governance
+model, and the Foundation has ratified new By-laws and charters, all projects,
+programs, and committees will continue to operate unchanged.
+
+Upon enactment of a new merged Foundation, or upon decision that the merger
+should not proceed, this working body will be disbanded.
+
+### Contributing to this repository
+
+Anyone may contribute to this repository via pull request. All individuals
+holding contributor status to any Node.js Foundation or JS Foundation project,
+unless voluntarily opting out, are considered members of the Bootstrap
+Committee and may request commit access to this repository.
+
+The [Code of Conduct][] explains the minimum behavior expectations required
+from all contributors.
+
+The Committee follows a [Consensus Seeking][] decision making model. Pull
+Requests must have at least two sign-offs from Committee members and no
+outstanding objections (or "Requests for Changes") in order to land. When there
+is disagreement on any particular issue, Committee members are encouraged to
+talk through the disagreement and come to a compromise resolution.
+
+The Committee does not vote. Should it become impossible to determine consensus
+on a given issue, each of the contending positions should be documented within
+the repository with the arguments for and against each clearly documented for
+further consideration.
```

Comment:
From your comments, what you suggest is that the final decisions will be made by the board/foundation staff.  If that is the case then I think you should state that here clearly.  ie if the bootstrap committee cannot come to a consensus then we document the divergent opinions and the board/foundation staff will make the decision.

Reply:
That is still not clear from the suggested text.